tests/Data/ProgrammesDb/EntityRepository/BroadcastRepository/FindUpcomingByServiceTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\BroadcastRepository;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\BroadcastRepository;
+use DateTimeImmutable;
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\BroadcastRepository::<public>
+ */
+class FindUpcomingByServiceTest extends AbstractDatabaseTest
+{
+    /** @var  BroadcastRepository */
+    private $repo;
+
+    public function setUp()
+    {
+        $this->loadFixtures(['BroadcastsFixture']);
+        $this->repo = $this->getEntityManager()->getRepository('ProgrammesPagesService:Broadcast');
+    }
+
+    public function testFindUpcomingByService()
```

Comment:
I like, but can we change the the name of the test(s)?, I think 
"function test< function name >" doesn't mean too much about what we are testing (in this way we just indicate the function we are hitting in our tests, but is not too useful). 
A testing can have different paths, and for each one we would need a test in order to test all the different behaviours. 

Thanks mate!